web/package.json

```diff
@@ -97,5 +97,9 @@
     "karma-notify-reporter": "^0.1.1",
     "karma-sourcemap-loader": "^0.3.5",
     "karma-webpack": "^1.7.0"
+  },
+  "devDependencies": {
+    "nodemon": "^1.7.0",
+    "webpack-dev-server": "^1.11.0"
```

Comment:
Right now when looking at the gulp file on would think you just need to run "gulp" to get everything running. People, like me, usually ignore the npm scripts and just go straight to the gulp file.